src/anm/media/audio.js

```diff
@@ -47,7 +47,7 @@ var audioContext = getAudioContext();
  * @class anm.Audio
  */
 function Audio(url) {
-    this.url = url + audioExt;
+    this.url = /\.\w+$/i.test(url) ? url : url + audioExt;
```

Comment:
yes, we will soon start exporting audio the same way (not really soon, but soon)